tasks/getHistoricalData.ts

```
@@ -85,6 +85,12 @@ task("getHistoricalData", "Retrieves the historical rates")
   )
   .addFlag("compound", "Get rates data from Compound")
   .addFlag("aave", "Get rates data from Aave")
+  .addOptionalParam(
+    "rate",
```

Should fall throw if borrow requested for unsupported platform, e.g. compound.